group_vars/moc/vlans.yaml

```diff
@@ -0,0 +1,940 @@
+vlans:
+  10:
+    name: "CSAIL-MAIN"
+    description: "Various Openshift clusters"
+  105:
+    name: "MOC-BU-PUBLIC"
+    description: "Public IPs from BU"
+  127:
+    name: "MOC-NEU-PUBLIC"
+    description: "NEU Public IP for infrastructure"
```

Comment:
VLAN 201 is missing. It's where the foreman instance is running and we still have a bunch of VMs and oVirt hosts on that network.